### ISSUE TITLE
Bootstrap 4 Theme Icon Issue in Chrome

### ISSUE THREAD

**Issue by msnyder-usu** [pytesNET/tail.select#82]
In Chrome, on the Bootstrap 4 theme, the arrows that are supposed to be on the far right with padding are instead on the far left. It seems that Chrome doesn't support the following: `background-position-x: right 0.75rem;`, specifically the second parameter isn't supported for some reason.

That said, an easy solution is the following CSS rule:
```
.select-label {
    background-position: center right .75rem;
}
```

**Comment by SamBrishes** [comment]
Hellow,

sorry for the long wait and thanks for your issue and the fix :D 

I copied this state from the Developer Tools of Firefox, unfortunately didn't checked back in Chrome.

Sincerely,
Sam.

**Comment by SamBrishes** [comment]
Has been fixed in 0.5.15.

Thanks!
